src/components/Note/NoteReactionList/NoteReactionBar/NoteReactionBar.scss

```diff
@@ -0,0 +1,60 @@
+@import "src/constants/style";
+
+.note-reaction-bar__root {
+  display: inline-flex;
+  flex-direction: row;
+  gap: $reaction__gap-size;
+
+  padding: $reaction__padding--outer-top-bottom $reaction__padding--outer-left-right $reaction__padding--outer-top-bottom 0; // left padding in NoteReactionList because of icon
+}
+
+.note-reaction-bar__reaction {
+  all: unset;
+
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+
+  width: $reaction__inner-size;
+  height: $reaction__inner-size;
+  padding: $reaction__padding--inner;
+
+  border-radius: $rounded--full;
+  font-size: $text--md;
+
+  cursor: pointer;
+
+  &:hover {
+    background-color: $hover-background-light;
```

Comment:
The hover state is barely visible. Maybe add a transform: scale() to the emojis similar to the hover state for changing the color of a column?